src/main/java/com/fasterxml/jackson/datatype/jsr310/ser/InstantSerializerBase.java

```diff
@@ -66,7 +77,7 @@ public void serialize(T instant, JsonGenerator generator, SerializerProvider pro
         }
         else
         {
-            generator.writeString(instant.toString());
+            generator.writeString(this.toIsoString.apply(instant));
```

Comment:
Please disregard this. I misunderstood in which file this change was taking place.

~~I don't understand why this change is necessary. According to the API documentation, `Instant#toString` already outputs a strictly ISO-compliant string.~~